src/restic/node.go

```diff
@@ -162,6 +170,26 @@ func (node Node) restoreMetadata(path string) error {
 		}
 	}
 
+	switch runtime.GOOS {
+	case "windows", "openbsd":
+		return nil
+	}
+	err = node.restoreAcls(path)
+	if err != nil {
+		debug.Log("error restoring acls for %v: %v", path, err)
+		return err
+	}
+
+	return nil
+}
+
+func (node Node) restoreAcls(path string) error {
```

Comment:
This function name is not a good fit, because it restores more than just ACL (think SELinux domain markers or other stuff). What about `restoreExtendedAttributes()` or `restoreAttributes()` (based on how you name the struct above, `Attribute` or `ExtendedAttribute`, please make it consistent)?